celesta-core/src/main/resources/ru/curs/celesta/score/celesta.sql

```diff
@@ -34,7 +34,7 @@
  */
 
 /**Celesta system grain. Not for modification.*/
-create grain celesta version '1.14';
+create grain celesta version '2.01';
```

Comment:
I  don't agree with this. Current version of `celesta` grain is 1.xx. So let it be as it is now, but with increment. We should not rely on other branches, that  haven't yet merged to master.

Reply:
Changed to 1.15